libcasr/src/asan.rs

```diff
@@ -85,6 +85,11 @@ impl ParseStacktrace for AsanStacktrace {
         // in function[(args)] [const] path
         // TODO: source file path may contain )
         if has_function {
+            if location.len() < 3 {
+                return Err(Error::Casr(format!(
+                    "Couldn't parse stack trace entry: {entry}"
```

Comment:
We shouldn't error. Instead, we should handle this corner case.

Reply:
I added an example to the tests and made minor changes to the regular expression above. This is an invalid frame and I don't know how to parse it correctly. Maybe we just need to discuss and refactor this parsing code later.